Documentation/Signoffs/SensorSysem.md

```diff
@@ -0,0 +1,106 @@
+# Sensor Subsystem Signoff
+## FUNCTION OF THE SUBSYSTEM
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The sensor subsystem is the actuator of our Air Quality design. These sensors will analyze and receive data from the outside and 
+send the new data from the interface straight to the microcontroller. Without this subsystem, the device we plan to design could never retrieve any real data for the 
+user to have and use. Therefore, the sensors are a crucial part of the process. For most gas sensors, they are propped up by a PCB board that allocates its data for 
+Master devices to store for data calculation.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; For experimental purposes, the design team has decided to get multiple types of sensors for the modular functionality to the device. 
+These types are digital, analog voltage, and analog current output sensors. While the digital sensors send bits describing the measurement of a specific parameter, the 
+analog sensors send their signals a bit differently. There are pros and cons on analog voltage or current signals in many regards. Voltage outputs for analog components
+are more common, but current signals are faster and less likely to lose current or voltage drops in the signal process. Current analog sensors have an extra step where 
+a built-in transmitter in the component takes the voltage and sets a current range for the signal to be generated. Once the current signal gets to the microcontroller, 
+it would need to go back to its voltage via a load resistor. The team plans on getting various types of sensors for the design process of this subsystem so that the 
+user of the Air Quality Device will be able to input various sensors that will accurately state the parameter value.
+
+## CONSTRAINTS ON THE SUBSYSTEM
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The subsystem constraints are placed to ensure that it can fulfill its job in a satisfactory manner.  The constraints given below 
+are generalized to allow for comparison of any potential contender for the sensors.  Constraints on the subsystem are as follows:
+
+1. Shall have a selection of tested sensors to cover a range of pollutants, particulates, air composites, and other qualities.  Common qualities to measure are carbon 
```

Comment:
Not specific enough. Exactly which items must be measured?

Hint: when writing the constraint list be thinking that the list will be given to *somebody else* and they must select components based on what is written.

Reply:
I have added some topics in the beginning of this section that the list touches on. This may give the reader some ideas of what to look for with other sensors